antora-ui-camel/src/css/catalog.css

```
@@ -0,0 +1,26 @@
+.catalog-container {
+  margin-top: 20px;
```

Always use `rem`, this is to help with different display devices and font customization in end user agents. 20px should be ~ 1.2rem (a bit more).

```suggestion
  margin-top: 1.2rem;
```